src/APIFunctions/2DPrinting.js

```diff
@@ -81,7 +76,8 @@ export function parseRange(pages, maxPages) {
  */
 export async function printPage(data) {
   let status = new ApiResponse();
-  await axios.post(PERIPHERAL_API_URL + '/Printer/sendPrintRequest', data)
+  await axios.post(PERIPHERAL_API_URL + '/Printer/sendPrintRequest',
```

Comment:
if possible manually undo all these unrelated changes, we can do this in a different eslint fix pr